libs-ts/cover-component/package.json

```diff
@@ -11,12 +11,9 @@
     "lint": "eslint .",
     "lint:fix": "eslint . --fix",
     "format": "just prettier",
-    "format:check": "just prettier-check",
-    "prepack": "bash ./prepack.sh",
-    "postpack": "bash ./postpack.sh"
```

Comment:
these were the remnants of the old gatsby cover optimizations, now moved to evans justfile